detox/.detoxrc.json

```diff
@@ -30,8 +30,8 @@
     "ios.simulator": {
       "type": "ios.simulator",
       "device": {
-        "type": "__DEVICE_TYPE__",
-        "os": "__DEVICE_OS__"
```

Comment:
Why the name changes?

Reply:
@larkox In the github actions, I have named it DEVICE_NAME for other workflows to pass any Device name and override. Wanted to keep local and ci with same reference to not get confused.